lib/logreader.c

```diff
@@ -329,9 +328,6 @@ log_reader_fetch_log(LogReader *self)
   gboolean may_read = TRUE;
```

Comment:
Would it be possible to also remove `may_read` ?

Reply:
It is a requirement of "multi-read" which is a used feature in `affile/linux-kmsg.c`.

Reply:
Yeah, you are right.